manual/parts/ipc.tex

```diff
@@ -207,7 +207,7 @@ \subsection{Errors}
 
     \item The destination slot cannot be looked up. Unlike the send
     system call, the \apifunc{seL4\_Recv}{sel4_recv} system call does not check that the
-    destination slot exists and is empty before it initiates the receive.
```

Comment:
Should this actually be left as ```receive```? I'm not sure ```reception``` is the correct technical term, but I'm not an expert :-)